liquibase-integration-tests/src/test/java/liquibase/dbtest/AbstractIntegrationTest.java

```diff
@@ -969,6 +971,21 @@ public void testInvalidIncludeDoesntBreakLiquibase() throws Exception {
         assertFalse(lockService.hasChangeLogLock());
     }
 
+    @Test
+    public void testInvalidSqlThrowsException() throws Exception {
```

Comment:
I am not familiar with this class. Does this run against snowflake?

Reply:
It runs to all configured databases on liquibase, but snowflake is not one of them. To test snowflake it needs to be on harness or liquibase-pro-tests.

Reply:
Nice. So I'll keep those tests (as they work for the other local databases) , but will log a debt to handle Snowflake .